cpp/environment/Registration.h

```diff
@@ -307,7 +313,7 @@ class RegisterBruteForce
         return makeVec3Matrix(T);
     }
 
-    double getRMSD()
+    float getRMSD() const
```

Comment:
There are many other places in this file where floating-point values are changed from using double to using float. What is the logic behind this?

Reply:
A lot of objects were being implicitly casted, e.g. a function returning `double` would be assigned to a `float`. In many places this was being done multiple steps removed from the initial place (e.g. in a member variable being assigned a map containing the output of a function returning double), but none of these cases that I could see had any reason to prefer doubles for numerical precision reasons (which Voronoi does and therefore explicitly chooses to use doubles for). For instance, if you trace the rmsd all the way back, it ultimately goes back to just adding a large number of `float`s together. In theory I suppose there is some precision loss, but I don't think there's any reason to expect that it's anything more than noise for the use case here. I changed everything to be a float to be consistent with the rest of freud, with the anticipation that implementing #464 would allow us to regain true double precision support if really necessary.